webroot/css/layouts/default.css

```diff
@@ -723,7 +723,7 @@ rt {
  }
 
 .language-icon {
-    box-shadow: 1px 1px 3px 0px rgba(204,204,204,1);
+    filter: drop-shadow(1px 1px 1.33px black);
```

Comment:
You changed the color of the shadow to `black`. Because of this, the shadow now looks much stronger than it used to be. Could you revert it to the original value `rgba(204,204,204,1)`?

Reply:
Thanks @jiru, please refer to https://github.com/Tatoeba/tatoeba2/pull/2334